client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -191,19 +195,30 @@ const InstalledInOrPricing = ( {
 	const translate = useTranslate();
 
 	if ( ( sitesWithPlugin && sitesWithPlugin.length > 0 ) || isWpcomPreinstalled ) {
+		/* eslint-disable wpcalypso/jsx-gridicon-size */
+		const isActive = !! activePlugins.find( ( activePlugin ) => activePlugin.slug === plugin.slug );
+
 		return (
-			/* eslint-disable wpcalypso/jsx-gridicon-size */
-			<div className="plugins-browser-item__installed">
-				<Gridicon icon="checkmark" size={ 14 } />
-				{ isWpcomPreinstalled || currentSites.length === 1
-					? translate( 'Installed' )
-					: translate( 'Installed on %d site', 'Installed on %d sites', {
-							args: [ sitesWithPlugin.length ],
-							count: sitesWithPlugin.length,
-					  } ) }
+			<div className="plugins-browser-item__installed-and-active-container">
+				<div className="plugins-browser-item__installed">
+					<Gridicon icon="checkmark" size={ 14 } />
+					{ isWpcomPreinstalled || currentSites.length === 1
+						? translate( 'Installed' )
+						: translate( 'Installed on %d site', 'Installed on %d sites', {
+								args: [ sitesWithPlugin.length ],
+								count: sitesWithPlugin.length,
+						  } ) }
+				</div>
+				{ currentSites.length === 1 && (
+					<div className="plugins-browser-item__active">
+						<Badge type={ isActive ? 'success' : 'info' }>
+							{ isActive ? 'Active' : 'Inactive' }
```

Comment:
When adding user-facing texts you need to use the `translate` function, otherwise, it won't be translated in other languages. More at: https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/README.md

```suggestion
							{ isActive ? translate( 'Active' ) :  translate( 'Inactive' ) }
```

Reply:
totally forgot to add translation! good catch here 👍🏼